mucgpt-frontend/src/index.css

```
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap");
```

Stylelint (17.11.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap"" (import-notation)

(import-notation)